### PR TITLE
Adds score display and submission functionality

### LINKED ISSUE #12: High scores sharing
Enable players to post their high scores to a server and check the top scores

### PR DESCRIPTION
**Pull request by jbernardo95**
Resolves #12

### CODE REVIEW

**Comment by jbernardo95** [comment]
@tiagonbotelho Review please 😉

File: lib/need_for_type/display_window.rb [hunk 1]
```diff
@@ -51,26 +51,52 @@ def render_game_text(text, chars_completed, stats, failed = false)
...
+      end
+    end
+    
+    def render_submit_score(username)
+      self.render_box do
+        self.set_render_pos(3, 4)
+        self.render_text("Submit your score !", GREEN, NORMAL)
+
+        self.set_render_pos(5, 4)
+        self.addstr("Type your username...")
```
Comment (tiagonbotelho) [inline lib/need_for_type/display_window.rb:93]: So we do not type our username haha 😛

**Review by tiagonbotelho — COMMENTED** [review]
looking good just left a comment otherwise I will merge

File: lib/need_for_type/display_window.rb (hunk 1 above)
Comment (jbernardo95) [reply lib/need_for_type/display_window.rb:93]: @tiagonbotelho No ?

**Pull request merged**